monitoring/uss_qualifier/scenarios/astm/utm/nominal_planning/nominal_planning.py

```
@@ -180,9 +181,7 @@ def _activate_first_flight(self):
             self.first_flight_activated,
         )
         if resp is None:
-            raise RuntimeError(
-                "Flight intent not activated successfully, but a High Severity issue didn't stop scenario execution"
-            )
+            return False
```

I think this should actually be a RuntimeError -- I think we should never be able to get a None resp here without triggering a High-severity failed check which will raise an exception that breaks out of the scenario (and therefore skips this line).  The exception here is to fail fast and loudly if that assumption isn't true.